Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -707,3 +707,381 @@
+  entries:
+  - name: bounty-item-chemVend
+    amount: 1
+    whitelist:
+      id:
+      - VendingMachineChemicals
+
+- type: pirateBounty
+  id: BountyVendingMachine
+  reward: 6
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-vendingmachine
+    amount: 3
+    whitelist:
+      id:
+      - VendingMachine
+
+- type: pirateBounty
+  id: BountyMercSuit
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-mercSuit
+    amount: 2
+    whitelist:
+      id:
+      - ClothingOuterHardsuitMercenary
+
+- type: pirateBounty
+  id: BountyCappy
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-cappy
+    amount: 10
+    whitelist:
+      id:
+      - MobCatCappy
+
+- type: pirateBounty
+  id: BountyCultistNecklace
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-cultistNeck
+    amount: 4
+    whitelist:
+      id:
+      - ClothingNeckAmuletBloodCult
+
+- type: pirateBounty
+  id: BountyNfsdCampaign
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-nfsdCampaign
+    amount: 4
+    whitelist:
+      id:
+      - ClothingHeadHatNfsdSmallCampaign
+      - ClothingHeadHatNfsdCampaign
+
+- type: pirateBounty
+  id: BountyScafSuit
+  reward: 5
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-scafsuit
+    amount: 3
+    whitelist:
+      id:
+      - ClothingOuterHardsuitScaf
+
+- type: pirateBounty
+  id: BountyRTG
+  reward: 4
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-rtg
+    amount: 2
+    whitelist:
+      id:
+      - GeneratorRTG
+
+- type: pirateBounty
+  id: BountyMercGas
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-mercgas
+    amount: 4
+    whitelist:
+      id:
+      - ClothingMaskGasMercenary
+
+- type: pirateBounty
+  id: BountyRDHardsuit
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-rdhardsuit
+    amount: 3
+    whitelist:
+      id:
+      - ClothingOuterHardsuitRd
+
+- type: pirateBounty
+  id: BountyMK58
+  reward: 5
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-mk58
+    amount: 2
+    whitelist:
+      id:
+      - WeaponPistolMk58
+
+- type: pirateBounty
+  id: BountyArtifactAnalyzer
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-artifactanalyzer
+    amount: 4
+    whitelist:
+      id:
+      - MachineArtifactAnalyzer
+
+- type: pirateBounty
+  id: BountyHotplate
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-hotplate
+    amount: 5
+    whitelist:
+      id:
+      - ChemistryHotplate
+
+- type: pirateBounty
+  id: BountyDeepFryer
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-deepfryer
+    amount: 4
```

Comment:
Deep fryer cannot even be bought on the flatpacker as far as im aware so its very rare to come around one and would need to steal 4 restaurant crews to even get one DB, it should be 2 deep fryers for 5DB.